.github/workflows/release.yml

```diff
@@ -58,5 +64,23 @@ jobs:
           version: ~> 1
           args: release --rm-dist
         env:
-          GITHUB_TOKEN: ${{ secrets.GH_PAT }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Are you sure this works? GoReleaser needs to push to supabase/homebrew-tap: https://github.com/goreleaser/goreleaser-action#limitation

Reply:
Good catch. Added a separate reference to `GH_PAT` in `.goreleaser`